recipes/parallelproj/meta.yaml

```
@@ -0,0 +1,63 @@
+{% set name = "parallelproj" %}
+{% set version = "1.2.6" %}
+{% set build_number = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/gschramm/parallelproj/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 826a83e51db6459004d17c030c21fb32e7cb5205a45c28c8f50f87f235321196
+
+build:
+  # Uncomment the following line if the package is pure Python and the recipe
+  # is exactly the same for all platforms. It is okay if the dependencies are
+  # not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python
+  # for more details.
+  # noarch: python
+
+  number: {{ build_number }}
+  string: "h{{ PKG_HASH }}_{{ build_number }}_cpu"   # [cuda_compiler_version == "None"]
+  string: "h{{ PKG_HASH }}_{{ build_number }}_cuda"  # [cuda_compiler_version != "None"]
+
+  # If the installation is complex, or different between Unix and Windows,
+  # use separate bld.bat and build.sh files instead of this key. By default,
+  # the package will be built for the Python versions supported by conda-forge
+  # and for all major OSs. Add the line "skip: True  # [py<35]" (for example)
+  # to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit
+  # to Windows.
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [cuda_compiler_version != "None"]
+    - cmake >=3.23
+    - ninja
+    - libgomp  # [linux]
+    - llvm-openmp  # [osx]
+  host:
+  run:
+    - libgomp  # [linux]
+    - llvm-openmp  # [osx]
```

It's strange to me that this is not a host dependency, but some build/run hybrid. It still works because `llvm-openmp`'s  run_export is [marked](https://github.com/conda-forge/openmp-feedstock/blob/main/recipe/meta.yaml#L32) "strong" (e.g. it exports even from the build env), and `libgomp` does so [too](https://github.com/conda-forge/ctng-compilers-feedstock/blob/main/recipe/meta.yaml#L446) (see [here](https://github.com/conda-forge/ctng-compilers-feedstock/blob/main/recipe/meta.yaml#L519)), but I'm almost certain that what you actually want is:
```suggestion
  host:
    - libgomp      # [linux]
    - llvm-openmp  # [osx]
```